From d5a9556c073367c9e07909bec2b5e2d753107909 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Fri, 25 Mar 2022 11:13:37 -0300
Subject: [PATCH 031/163] crypt: Remove unused variable on cert test

Checked on x86_64-linux-gnu and i686-linux-gnu.

(cherry picked from commit 1dedc5bd5cc43173e0f7d7e010398ff46139e1dd)
---
 crypt/cert.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/crypt/cert.c b/crypt/cert.c
index e070ca398d..32c4386caf 100644
--- a/crypt/cert.c
+++ b/crypt/cert.c
@@ -50,10 +50,9 @@ main (int argc, char *argv[])
 {
 	char key[64],plain[64],cipher[64],answer[64];
 	int i;
-	int test;
 	int fail;
 
-	for(test=0;!feof(stdin);test++){
+	for(;!feof(stdin);){
 
 		get8(key);
 		printf(" K: "); put8(key);
-- 
2.37.2

